.github/workflows/publish-prerelease.yml

```diff
@@ -73,14 +76,14 @@ jobs:
       - name: Archive Binary
         uses: actions/upload-artifact@v3
         with:
-          name: ceramic-one_${{ matrix.target }}
-          path: ceramic-one_${{ matrix.target }}.bin.tar.gz
+          name: ceramic-one_${{ matrix.target }}-${{ matrix.os }}
+          path: ceramic-one_${{ matrix.target }}-${{ matrix.os }}.bin.tar.gz
       - name: Archive Debian Package
         uses: actions/upload-artifact@v3
         if: ${{ matrix.ext == 'deb' }}
         with:
-          name: ceramic-one_${{ matrix.target }}
-          path: ceramic-one_${{ matrix.target }}.tar.gz
+          name: ceramic-one_${{ matrix.target }}-${{ matrix.os }}
+          path: ceramic-one_${{ matrix.target }}-${{ matrix.os }}.tar.gz
```

Comment:
We'll also need to use the new OS suffix in `ci-scripts/package.sh` when preparing the archive, and in `.github/workflows/update-homebrew.yml` when retrieving the archive SHA.

Reply:
Actually, wait, let me check that again.

Reply:
Yes, I think we'll need to update the other two files since the specific OS version is now also part of the archive name.

Reply:
As well as [this](https://github.com/ceramicnetwork/js-ceramic/blob/develop/ci-scripts/install-rust-ceramic.sh#L20) `js-ceramic` file.